components/Layout/Layout.js

```diff
@@ -0,0 +1,14 @@
+import Footer from "../Footer";
+import SiteHeader from "../SiteHeader";
+
+const Layout = ({ title, children }) => {
+  return (
+    <>
+      <SiteHeader />
```

Comment:
Maybe we can use this `Head` component from Next.js to do the title part here: https://nextjs.org/docs/api-reference/next/head